woocommerce/woocommerce-gutenberg-products-block
# Add Playwright tests for All Reviews, Reviews by Product and Reviews by Category blocks

## What

~Blocked by https://github.com/woocommerce/woocommerce-blocks/pull/11976.~

Fixes https://github.com/woocommerce/woocommerce/issues/42065.

This PR adds Playwright tests to the All Reviews, Reviews by Product and Reviews by Category blocks and removes the Puppeteer tests for the same blocks.

## Why

Hopefully, these tests would have caught these issues: https://github.com/woocommerce/woocommerce-blocks/issues/11912, https://github.com/woocommerce/woocommerce-blocks/issues/11917 and https://github.com/woocommerce/woocommerce-blocks/issues/11974.

## Testing Instructions

_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Verify Playwright e2e tests pass.


## Screenshots or screencast

| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility

Required:


## Checklist

Required:


Conditional:


## Changelog


> Add suggested changelog entry here.
